src/routes/api/collabs/[id]/exportassets.ts

```diff
@@ -0,0 +1,99 @@
+import type { RequestHandler } from '@sveltejs/kit';
+import cookie from 'cookie';
+import { Jwt } from '../../../../jwt';
+import { Prisma } from '../../../../database/prisma';
+import { DiscordUser } from '../../../../utils/discord/user';
+import { ServerPaths } from '../../../../utils/paths/server';
+import { readFileSync } from 'fs';
+import path from 'path';
+import JSZip from 'jszip';
+
+export const get: RequestHandler = async ({ request, params }) => {
+	const cookieHeader = request.headers.get('cookie');
+	const cookies = cookie.parse(cookieHeader ?? '');
+	const decoded = Jwt.decode(cookies['discord_token']);
+	const token = decoded['access_token'] as string;
+
+	const decodedUser = Jwt.decode(cookies['user_id']);
+	const userId = decodedUser['user_id'] as string;
+
+	const collab = await Prisma.client.collab.findUnique({
+		where: {
+			id: params.id
+		},
+		include: {
+			collabAssets: true
+		}
+	});
+
+	if (!collab) {
+		return {
+			status: 404,
+			body: {
+				message: 'Collab not found'
+			}
+		};
+	}
+
+	const user = await DiscordUser.getUser(userId, token);
+
+	if (!user || !user.admin || userId !== '687004886922952755') {
+		return {
+			status: 403
+		};
+	}
+
+	const participants = await Prisma.client.participant.findMany({
+		orderBy: [
+			{
+				createdAt: 'asc'
+			}
+		],
+		where: {
+			collabId: collab.id
+		},
+		include: {
+			pick: {
+				include: {
+					assets: {
+						include: {
+							collabAsset: true
```

Comment:
Considering the size of this query, it's better to query picks directly. We don't need the participant relation for the export.